src/components/TodoList/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TodoList';
```

Comment:
The exports are done directly from an import, which is not recommended. Please import the module and then export it separately. 🛠